Formula/apache-flink.rb

```diff
@@ -1,25 +1,55 @@
 class ApacheFlink < Formula
   desc "Scalable batch and stream data processing"
   homepage "https://flink.apache.org/"
-  url "https://www.apache.org/dyn/closer.lua?path=flink/flink-1.3.2/flink-1.3.2-bin-hadoop27-scala_2.11.tgz"
-  version "1.3.2"
-  sha256 "378be2346990264eeff2ae012175523fc94ddbe59bead7e2a76e0d93c0c7c455"
+  url "https://www.apache.org/dyn/closer.lua?path=flink/flink-1.5.0/flink-1.5.0-bin-hadoop27-scala_2.11.tgz"
```

Comment:
any reason not to move to hadoop28?

Reply:
I decided for Hadoop 2.7 because I wanted to avoid excluding people who have an older Hadoop version installed. But with this argument one would have to provide Flink with Hadoop 2.0, I guess. I would also be ok with upgrading to Hadoop 2.8.

I also thought about providing the Hadoop free Flink version, but then one would have to export the `HADOOP_CLASSPATH` manually before running Flink on Yarn, for example. Here my concern was that it might throw people off when using Flink for the first time.

Reply:
I'm fine with waiting for a user to request the move to 2.8 if you think 2.7 is preferable for now.